platforms/web/index.html

```diff
@@ -2,16 +2,30 @@
 <html lang="en">
 <head>
   <meta charset="utf-8" />
-  <title>JW OTT Webapp</title>
+  <title><% name %></title>
   <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-  <meta name="description" content="JW OTT Webapp is an open-source, dynamically generated video website." data-react-helmet="true" />
+  <meta name="description" content="<% description %>" data-react-helmet="true" />
   <meta name="twitter:card" content="summary" data-react-helmet="true"  />
-  <meta property="og:title" content="JW OTT Webapp" data-react-helmet="true" >
-  <meta property="og:description" content="JW OTT Webapp is an open-source, dynamically generated video website." data-react-helmet="true">
-  <meta property="og:image" content="/images/icon-256x256.png" data-react-helmet="true">
+  <meta property="og:title" content="<% name %>" data-react-helmet="true" >
+  <meta property="og:description" content="<% description %>" data-react-helmet="true">
+  <meta property="og:image" content="/images/icons/app-icon.png" data-react-helmet="true">
 
-  <link rel="icon" href="/favicon.ico" type="image/x-icon" />
+  <link rel="icon" href="/images/icons/favicon.ico" type="image/x-icon" sizes="any">
+  <link rel="icon" type="image/png" sizes="196x196" href="/images/icons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="160x160" href="/images/icons/favicon-64x64.png">
+  <link rel="icon" type="image/png" sizes="96x96" href="/images/icons/favicon-96x96.png">
+  <link rel="icon" type="image/png" sizes="64x64" href="/images/icons/favicon-64x64.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/images/icons/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/images/icons/favicon-16x16.png">
+  <link rel="apple-touch-icon" href="/images/icons/apple-touch-icon-180x180.png">
+  <link rel="apple-touch-icon" href="/images/icons/apple-touch-icon-152x152.png">
+  <link rel="apple-touch-icon" href="/images/icons/apple-touch-icon-144x144.png">
```

Comment:
Do we need these icons in case we already generate them automatically using assets-generator?

Reply:
Can they also be auto-injected?

Reply:
@langemike do you know if auto-injecting the icons is possible as well?

Reply:
Unfortunately the asset-generator has a shortcomings to create the correct link to the icons, because of the monorepo structure. Though we can inject the tags based on our own defined settings. I created this PR for it https://github.com/Videodock/ott-web-app/pull/126